pkg/controller/namespace/namespace_controller_test.go

```diff
@@ -0,0 +1,137 @@
+package namespace
+
+import (
+	"context"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	realclient "sigs.k8s.io/controller-runtime/pkg/client"
+	client "sigs.k8s.io/controller-runtime/pkg/client/fake"
```

Comment:
Your deps need to be updated. This is pulling in stuff that's not present in your vendor.

Reply:
I think I got it now. Can you give it a go?

Reply:
It works for me after `git clean -xdf` and a clean checkout (and fixing check errors)